src/main/kotlin/com/developerphil/adbidea/ui/NotificationHelper.kt

```diff
@@ -1,20 +1,54 @@
 package com.developerphil.adbidea.ui
 
-import com.intellij.notification.NotificationDisplayType
 import com.intellij.notification.NotificationGroup
+import com.intellij.notification.NotificationGroupManager
+import com.intellij.notification.NotificationListener
 import com.intellij.notification.NotificationType
 
 object NotificationHelper {
-    private val INFO = NotificationGroup("ADB Idea (Logging)", NotificationDisplayType.NONE, true, null, null)
-    private val ERRORS = NotificationGroup("ADB Idea (Errors)", NotificationDisplayType.BALLOON, true, null, null)
+    fun info(message: String) {
+        sendNotification(
+            message,
+            NotificationType.INFORMATION,
+            NotificationGroupManager
+                .getInstance()
+                .getNotificationGroup("ADB Idea (Logging)")
+        )
+    }
+
+    // Function to send an error notification
+    fun error(message: String) {
+        sendNotification(
+            message,
+            NotificationType.ERROR,
+            NotificationGroupManager
+                .getInstance()
+                .getNotificationGroup("ADB Idea (Errors)")
+        )
+    }
 
-    fun info(message: String) = sendNotification(message, NotificationType.INFORMATION, INFO)
+    // Helper function to create and display a notification
+    private fun sendNotification(
+        message: String,
+        notificationType: NotificationType,
+        notificationGroup: NotificationGroup,
+    ) {
+        // Create the notification without a listener
+        val notification = notificationGroup.createNotification(
+            "ADB IDEA",
+            escapeString(message),
+            notificationType
+        )
 
-    fun error(message: String) = sendNotification(message, NotificationType.ERROR, ERRORS)
+        // Set the listener separately, if needed
+        notification.setListener(NotificationListener.URL_OPENING_LISTENER)
```

Comment:
This method is deprecated. When does the listener need to be set separately?

Reply:
![image](https://github.com/user-attachments/assets/90329120-dd0e-45de-8342-2d50c65cf7d6)

We have two options: provide a Notification Listener in createNotification() or set it separately using setListener(). 

createNotification() is scheduled for removal, but setListener() is still marked as deprecated.

However, from the look at the project, we don't need listeners as we are not providing any links in the Notification. I'm removing the listener from both usages. Please let me know if you'd like them to be there.